content/index.rst

```diff
@@ -21,144 +28,97 @@ dynamic_title:
-    with full transparency and daily communication.
-  - **Because we're experts**,
-    we can guide you smoothly through the `entire process`_,
-    navigating issues before they arise.
-  - **Because we integrate** `design and code and content`_,
-    our software is built to last,
-    keeping your maintenance costs low.
-
-  .. _`team of designers & developers`: /birds/
-  .. _entire process: /services/planning/
-  .. _design and code and content: /services/
-
-  We know you have options,
-  and we'd love to talk more.
-  **We think you'll be happy with the results.**
-
-.. callmacro:: content.macros.j2#get_quotes
-  :page: 'work/coachhub'
-  :slug: 'remote'
-
-.. callmacro:: projects/splash.macros.j2#splash_list
-  :headline: 'Featured Projects'
-  :url: '/work/'
 
-.. callmacro:: content.macros.j2#divider
-  :title: 'Human-Driven Design'
+  I co-founded OddBird with my brothers in 2008
+  to help you create **scalable web applications**
+  with **human-centered design**,
+  **performance**,
+  and **accessibility**.
 
-.. callmacro:: birds/macros.j2#home_faces
-  :authors: ['carl', 'kit', 'stacy', 'miriam', 'sondra', 'jonny']
-
-.. callmacro:: content.macros.j2#image_block
-  :image: '/static/images/projects/trig-mobile.jpg'
-  :headline: 'Are We The Right Fit For You?'
-
-  **Software is successful
-  when it solves real problems in people's lives.**
-  We can help you find those solutions --
-  with 100% test coverage,
-  robust architecture,
-  and living style guides.
-  *Enjoy low on-going maintenance,
-  and significant savings for years.*
+  From custom builds to consulting and refactors:
+  we'll provide an established team of experts to fit your needs.
+  Our well-tested and documented code will keep
+  **maintenance costs low, for significant long-term savings**.
+  We think you’ll be happy with the results.
 
   .. callmacro:: content.macros.j2#link_button
     :url: '/contact/'
 
-    Start the conversation
+    Let's talk about your project
 
-.. callmacro:: content.macros.j2#get_quotes
-  :page: 'work/coachhub'
-  :slug: 'handoff'
+.. callmacro:: content.macros.j2#two_quotes
+  :quotes: [
+      ['work/coachhub', 'remote'],
+      ['work/medcurbside', 'goals'],
+    ]
 
-.. callmacro:: content.macros.j2#divider
+.. callmacro:: projects/splash.macros.j2#splash_list
+  :headline: 'Industries We Serve'
+  :url: '/work/'
+
+.. callmacro:: birds/macros.j2#home_faces
+  :authors: ['carl', 'kit', 'stacy', 'miriam', 'sondra', 'jonny']
 
-.. callmacro:: content.macros.j2#get_quotes
-  :page: 'work/medcurbside'
-  :slug: 'goals'
+.. callmacro:: content.macros.j2#two_quotes
+  :quotes: [
+      ['work/timedesigner', 'innovative'],
+      ['work/coachhub', 'handoff'],
+    ]
 
 .. callmacro:: content.macros.j2#image_block
   :image: '/static/images/pages/jssass.png'
   :url: 'https://www.sitepoint.com/premium/books/jump-start-sass'
-  :headline: 'Industry-Leading Expertise'
+  :headline: 'Python, CSS, & Performance Experts'
```

Comment:
A question: should we be highlighting JS front-end expertise more?

I'm not asking because I'm personally offended or anything... but I notice that two of our current/potential clients (SFDO and Aunalytics) were specifically looking for JS expertise. 🤷‍♂️

Reply:
yeah, I had a similar question. I'd love to discuss this section a bit more

Reply:
@SondraE @jgerigmeyer what do you think about `Performant Python, JavaScript, & CSS`?

Reply:
Sure, I think that works. The location of "Accessible HTML" in this list surprises me a bit -- is that something potential clients are looking for?

Reply:
I think "accessibility" and "performance" are things people look for, but they maybe don't belong in this list…

Reply:
@mirisuzanne I was copying and pasting "accessible HTML" from the list on Application Design & Development page in the paragraph at the top. Feel free to remove if it doesn't fit in the home page list.